src/transform.ts

```diff
@@ -0,0 +1,18 @@
+import type { Options, TransformOutput } from "../lib/wasm";
+import swc from "../lib/wasm.js";
+
+const DEFAULT_OPTIONS: Options = {
+	mode: "strip-only",
+};
+
+export function transformSync(
+	source: string,
+	options: Options,
+): TransformOutput {
+	// Ensure that the source code is a string
+	const input = !source || typeof source !== "string" ? "" : source;
```

Comment:
why not throw an error if not good type intend of putting empty string ?

Reply:
because it happens in node that the source is undefined

Reply:
being discussed here https://github.com/nodejs/amaro/pull/45